discovery-provider/src/api/v1/users.py

```diff
@@ -393,6 +394,40 @@ def get(self, user_id):
         favorites = list(map(extend_favorite, favorites))
         return success_response(favorites)
 
+
+tags_route_parser = reqparse.RequestParser()
+tags_route_parser.add_argument('user_id', required=False, type=str)
+tags_route_parser.add_argument('limit', required=False, type=int)
+tags_response = make_response("tags_response", ns, fields.List(fields.String))
+@ns.route("/<string:user_id>/tags")
+class FavoritedTracks(Resource):
+    @record_metrics
+    @ns.doc(
+        id="""Get User's Most Used Track Tags""",
+        params={
+            'user_id': 'A User ID',
+            "limit": 'Limit on the number of tags'
+        },
+        responses={
+            200: 'Success',
+            400: 'Bad request',
+            500: 'Server error'
+        }
+    )
+    @full_ns.expect(tags_route_parser)
+    @ns.marshal_with(tags_response)
+    @cache(ttl_sec=5)
```

Comment:
Let's up the caching on this one. We don't need it to be super fresh. I would say at least a minute